app/views/layouts/_left_nav.html.haml

```diff
@@ -0,0 +1,12 @@
+%div{ class: 'small-2 columns' }
+  .row
+    %div{ class: 'small-12 medium-10 large-8 columns left_nav' }
+      %ul{ class: 'vertical menu'}
+        %li.heading Students
+        .student_list= render @students
+        %div{ class: "add_student link" }= link_to "Add Student", new_student_path
```

Comment:
Line is too long. [82/80]<br>Avoid defining `class` in attributes hash for static class names